docs/sources/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -107,6 +107,53 @@ metaMonitoring:
             app.kubernetes.io/name: kube-state-metrics
 ```
 
+#### Sending metrics back into Mimir/GEM
+
+You can also send the collected metamonitoring metrics to the installation of Mimir or GEM.
+The configuration varies slightly for GEM and Mimir.
+
+If you have deployed Mimir, then the Helm chart values should look like the following example.
+Replace `HELM_RELEASE_NAME` with the name of your Helm release:
+
+```yaml
+metaMonitoring:
+  serviceMonitor:
+    enabled: true
+  grafanaAgent:
+    enabled: true
+    installOperator: true
+
+  metrics:
+    remote:
+      url: "http://<HELM_RELEASE_NAME>-mimir-nginx.mimir.svc/api/v1/push"
```

Comment:
It's going to be just mimir-nginx if the release name is "mimir" due to that shortcut in helpers. I suggest to just say the internal write endpoint as reported by helm install/upgrade

Reply:
If we allowed template expansion here, we could write
```
url: '{{ include "mimir.gatewayUrl" $ }}
```
maybe?